msg/versioned/ArmingCheckReply.msg

```diff
@@ -1,34 +1,38 @@
+# Arming check reply.
+#
+# Response to an ArmingCheckRequest message.
+
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp # time since system start (microseconds)
+uint64 timestamp # Time since system start [us]
 
-uint8 request_id
-uint8 registration_id
+uint8 request_id # Id of ArmingCheckRequest for which this is a response.
+uint8 registration_id # Id of component emitting this response.
 
-uint8 HEALTH_COMPONENT_INDEX_NONE = 0
+uint8 HEALTH_COMPONENT_INDEX_NONE = 0 # Index of health component for which this response applies.
 
-uint8 health_component_index      # HEALTH_COMPONENT_INDEX_*
-bool health_component_is_present
-bool health_component_warning
-bool health_component_error
+uint8 health_component_index # HEALTH_COMPONENT_INDEX_*
+bool health_component_is_present ?
+bool health_component_warning # ?
+bool health_component_error # ?
```

Comment:
What are the health components? 
My guess is that perhaps these map to the https://mavlink.io/en/messages/common.html#MAV_SYS_STATUS_SENSORS. But then there are multiple sensors, so if a registration is a mode, you'd have to send this for every sensor???
How does this work?

Reply:
This is currently unused, but goes together with the [health events interface](https://github.com/PX4/PX4-Autopilot/blob/b98d0d44067b75bec480fe3f4657a531d0df0cad/src/lib/events/enums.json#L148)